src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -313,6 +333,8 @@ protected Response execute(final HttpUrl url, final String method, final Object
      * @param response Response to parses the error details from
      */
     protected void parseAndThrowErrorResponse(final Response response) {
+        logger.debug(LOG_HTTP_RESPONSE_BODY_TMPL, gson.toJson(response.body()));
```

Comment:
With a little more context, such as ERROR Failed to parse error message, response body received: XXXXXXXXXX